regtests/bin/matrix_ncep

```diff
@@ -180,44 +180,46 @@
-  export    oasis='y' # Atmosphere, ocean, and ice coupling using OASIS
-  export calendar='y' # Calendar type
-  export confignc='y' # Configurable netCDF meta data (ww3_ounf)
-
-  export  multi01='y' # mww3_test_01 (wetting and drying)
-  export  multi02='y' # mww3_test_02 (basic two-way nesting test))
-  export  multi03='y' # mww3_test_03 (three high and three low res grids).
-  export  multi04='y' # mww3_test_04 (swell on sea mount and/or current)
-  export  multi05='y' # mww3_test_05 (three-grid moving hurricane)
-  export  multi06='y' # mww3_test_06 (curvilinear grid tests)
-  export  multi07='y' # mww3_test_07 (unstructured grid tests)
-  export  multi08='y' # mww3_test_08 (wind and ice tests)
-  export  multi09='y' # mww3_test_09 (SMC multi grid test)
-  export      ufs='y' # The Unified Forecast System
-  export     grib='y' # grib file field output
-  export rstrt_b4b='y' # Restart Reproducibility
-  export  npl_b4b='y' # MPI task Reproducibility
-  export  nth_b4b='y' # Thread Reproducibility
-  export     esmf='n' # ESMF coupling
+  export       shrd='y' # Do shared architecture tests
+  export       dist='y' # Do distributed architecture (MPI) tests
+  export        omp='y' # Threaded (OpenMP) tests
+  export       hybd='y' # Hybrid options
+
+  export     prop1D='y' # 1-D propagation tests (ww3_tp1.X)
+  export     prop2D='y' # 2-D propagation tests (ww3_tp2.X)
+  export       time='y' # time linmited growth
+  export      fetch='y' # fetch linmited growth
+  export     hur1mg='y' # Hurricane with one moving grid
+  export      shwtr='y' # shallow water tests
+  export      unstr='y' # unstructured grid tests
+  export      pdlib='y' # unstr with pdlib for domain decomposition and implicit solver
+  export      smcgr='y' # SMC grid test
+  export        rtd='y' # Rotated pole test
+  export     mudice='y' # Mud/Ice and wave interaction tests 
+  export     infgrv='y' # Second harmonic generation tests
+  export       uost='y' # ww3_ts4 Unresolved Obstacles Source Term (UOST)
+  export      assim='y' # Restart spectra update
+  export      oasis='y' # Atmosphere, ocean, and ice coupling using OASIS
+  export   calendar='y' # Calendar type
+  export   confignc='y' # Configurable netCDF meta data (ww3_ounf)
+
+  export    multi01='y' # mww3_test_01 (wetting and drying)
+  export    multi02='y' # mww3_test_02 (basic two-way nesting test))
+  export    multi03='y' # mww3_test_03 (three high and three low res grids).
+  export    multi04='y' # mww3_test_04 (swell on sea mount and/or current)
+  export    multi05='y' # mww3_test_05 (three-grid moving hurricane)
+  export    multi06='y' # mww3_test_06 (curvilinear grid tests)
+  export    multi07='y' # mww3_test_07 (unstructured grid tests)
+  export    multi08='y' # mww3_test_08 (wind and ice tests)
+  export    multi09='y' # mww3_test_09 (SMC multi grid test)
+
+  export        ufs='y' # The Unified Forecast System
+  export  ufscoarse='y' # Option for small PCs
```

Comment:
This option needs to be added to all matrix_* files and not just matrix_ncep.

Reply:
done